starboard/win/shared/BUILD.gn

```diff
@@ -30,16 +30,18 @@ config("starboard_platform_config") {
 
   cflags = [
     "/EHsc",  # C++ exceptions (required with /ZW)
-  ]
-
-  cflags += [
     "/FU${msvc_path}/lib/x86/store/references/platform.winmd",
     "/FU${windows_sdk_path}/References/$wdk_version/Windows.Foundation.FoundationContract/4.0.0.0/Windows.Foundation.FoundationContract.winmd",
     "/FU${windows_sdk_path}/References/$wdk_version/Windows.Foundation.UniversalApiContract/14.0.0.0/Windows.Foundation.UniversalApiContract.winmd",
     "/FU${windows_sdk_path}/References/$wdk_version/Windows.UI.ViewManagement.ViewManagementViewScalingContract/1.0.0.0/Windows.UI.ViewManagement.ViewManagementViewScalingContract.winmd",
-    "/FU${windows_sdk_path}/References/$wdk_version/Windows.Xbox.ApplicationResourcesContract/2.0.0.0/Windows.Xbox.ApplicationResourcesContract.winmd",
-    "/FU${windows_sdk_path}/References/$wdk_version/Windows.Xbox.Security.ApplicationSpecificDeviceAuthenticationContract/1.0.0.0/Windows.Xbox.Security.ApplicationSpecificDeviceAuthenticationContract.winmd",
   ]
+
+  if (is_internal_build) {
+    cflags += [
```

Comment:
Why is this in shared win code instead of xb1 specific code?

Reply:
I don't know about the original decision to put it here, but I can move it to starboard/xb1/BUILD.gn. I see there's currently cflags_cc defined there, but not cflags. In gn would I simply add

```
if (is_internal_build) {
  cflags = [ <dependencies> ]
}
```

or would I need to make it `cflags +=` to avoiding overriding anything in this file.

Reply:
I'm not sure what you mean about overriding cflags, as cflags in one target wouldn't override cflags in another target. Otherwise, it sounds like what you said would work.

I am concerned by the use of a winuwp conditional in https://github.com/youtube/cobalt/blob/main/starboard/win/shared/BUILD.gn#L20-L23, as I believe winuwp flags should ideally only be applied to uwp/ files (or slightly worse, globally), but both of these cleanups are tangential to what's going on here so we don't have to figure it all out in this PR